index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
```

Comment:
We should probably not passthrough `opts` to hyperkv here. The only `opt` that `hyperkv.get()` takes is `opts.fields` which if passed will break the following logic.